src/main/java/website/frontrow/util/CollisionHandler.java

```diff
@@ -14,8 +11,9 @@
 import website.frontrow.logger.Logable;
 
 /**
- * This Class is for handling collisions during the game.
+ * Computes collisions during the game.
  */
+// TODO: Refactor name to CollisionComputer
```

Comment:
I was actually waiting for your changes to be pushed. So I guess I'll merge dev into my branch and fix that.